tests/unit/App.spec.ts

```diff
@@ -980,7 +919,7 @@ describe('Restoration - App page', () => {
   afterEach(() => {
     window.location.assign = assign
     sinon.restore()
-    vi.resetAllMocks()
+    // vi.resetAllMocks() // leave mocks for next 2 test suites
```

Comment:
The alternative was to duplicate the mocks from this test suite to the test suites below. This works. If you hate it, lmk.